solution

```diff
@@ -0,0 +1,120 @@
+/* ----------------------------------------------------------------------------
+* Copyright &copy; 2015 Duy Nguyen <andy21996@csu.fullerton.edu>
+* Released under the [MIT License] (http://opensource.org/licenses/MIT)
+* ------------------------------------------------------------------------- */
+
+
+#include "stdafx.h"
+#include <iostream>
+#include <vector>
+#include <ostream>
+using std::cout;
+using std::endl;
+using std::vector;
+
+void print_array(int size, const int array[])
+{
+	cout << "( ";
+	for (int i = 0; i < size; i++)
+		cout << array[i] << " ";
+	cout << " )";
+}
+
+void selection_sort_in_place(int size, int array[])
+{
+	for (int currentIndex = 0; currentIndex < size - 1; currentIndex++)
+	{
+		int minIndex = currentIndex + 1;
+		int minValue = array[minIndex];
+		for (int i = currentIndex + 1; i < size; i++)
+		{
+			if (array[i] < minValue)
+			{
+				minValue = array[i];
+				minIndex = i;
+			}
```

Comment:
this should work :-) though, it's a bit redundant to store both the `minIndex` and the `minValue` as local variables